### DISCUSSION TITLE
Pole Figure of Rolls Royce Angles in MTEX

### DISCUSSION THREAD

**Discussion by mariuslueck** [mtex-toolbox/mtex#1946] [Ask Anything]
Hello,

I have the 6 angles according to the Rolls Royce convention (theta, phi, kappa, alpha, delta and gamma) from a Laue measurement. I was wondering if MTEX could give me a pole figure using the angles. I have not found anything in the documentation, a translation to Euler angles according to Bunge would also be helpful.

Thanks a lot!

**Comment by ralfHielscher** [comment]
Do you have any more information about the  Rolls Royce convention?

Ralf.

**Reply by mariuslueck** [reply]
Yes

https://www.researchgate.net/publication/258762531_Zur_Orientierungsbestimmung_von_Einkristallen
